### ISSUE TITLE
The model did not behave well with my dataset

### ISSUE THREAD

**Issue by 1999EMMANUEL** [BFeng14/ActFound#7]
Hello, I have a dataset, but the amount of data is quite limited, and the prediction results on ActFound are not very good. Do I have to give up using ActFound? Are there any other ways to improve the predictive power for my task? Thank you.

**Comment by BFeng14** [comment]
Hi, thank you very much for your interest on Actfound, Actfound is a bioactivity foundation model trained on ChEMBL and BindingDB, and it is normal to perform poorly on certain dataset. Can you share me what type of assay your are testing on, and the number of your finetuning compounds? which might be helpful for me to provide more useful suggestions.

**Comment by 1999EMMANUEL** [comment]
Thank you for your reply，It is a set of data regarding the intracellular accumulation of compounds, with just over 70 entries. I have normalized the values, but no matter how I adjust the hyperparameters and test set, the requirements are not met. It's possible that this property is not well compatible with biological activity.

**Comment by BFeng14** [comment]
Hi, I think there are two things that you can check: 
1. in the training phase, the activity=pX=-logX (e.g. pIC50=-logIC50), and higher value means better activity, you can check the value meaning of your test dataset to make consistent with our training dataset. 
2. you can check the unit of you data, if it is mol, kg/L, mg/kg or %, your activity should be in log scale. you can check the value ranges of your dataset, if it covers multiple orders of magnitude and the errors of your experiments are expressed as multiples (e.g. 3 times), then you should use log scale.

**Comment by 1999EMMANUEL** [comment]
Thank you very much for your advice
